W-15946540 AEH June ga content pr jh (#38)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/setting-up-the-api-portal.adoc

= Setting Up the API Portal 
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: styling-your-api-portal.adoc

> After connecting to your Salesforce organization, the guided setup takes portal administrators through the process setting up the portal. The setup process enables administrators to choose how to create the portal and style it using out-of-the-box templates.

Reviewer: can you use "you" instead of portal administrators?

> After connecting to your Salesforce organization, the guided setup takes portal administrators through the process setting up the portal. The setup process enables administrators to choose how to create the portal and style it using out-of-the-box templates.

Reviewer: ```suggestion
After connecting to your Salesforce organization, the guided setup takes portal administrators through the process setting up the portal, including creating and styling it using out-of-the-box templates.
```

> You can create a new portal, use an existing portal to create the new portal, or create the portal later. The option to create a new portal uses an existing portal that was disconnected from a Salesforce organization. The new portal uses the styling metadata from the existing portal. Selecting this option from the guided setup enables you to quickly set up the portal without going through the styling process. Since you can have only one Salesforce organization per portal, the existing portal is overwritten and the new portal is created for that Salesforce organization.

Reviewer: This is confusing to me. I thought it was three options: new, new from existing, or wait. But this seems to imply otherwise:
The option to create a new portal uses an existing portal that was disconnected from a Salesforce organization. The new portal uses the styling metadata from the existing portal.

Author: Yes, there are 3 ways. I didn't add creating a new portal to this sentence because it seems self-explanatory (create a new portal).

> You can create a new portal, use an existing portal to create the new portal, or create the portal later. The option to create a new portal uses an existing portal that was disconnected from a Salesforce organization. The new portal uses the styling metadata from the existing portal. Selecting this option from the guided setup enables you to quickly set up the portal without going through the styling process. Since you can have only one Salesforce organization per portal, the existing portal is overwritten and the new portal is created for that Salesforce organization.

Reviewer: ```suggestion
You can create a new portal, use an existing portal to create the new portal, or create the portal later. The option to create a new portal uses an existing portal that was disconnected from a Salesforce organization. The new portal uses the styling metadata from the existing portal. Selecting this option from the guided setup enables you to quickly set up the portal without going through the styling process. Because you can have only one Salesforce organization per portal, the existing portal is overwritten with the new portal in the Salesforce organization.
```

Author: You don't create a portal in a Salesforce organization but one for the SF org.

Author: I updated this paragraph with your suggestions but kept ..."for that Salesforce organization".

> The guided design contains basic styling options for the Home, API Catalog, and API Details pages with a live preview to help you visualize your portal. Each page includes a progress bar to see where you're in the portal design process. 

Reviewer: Do you need this paragraph?

Author: Keeping it since it explains the image that you see is a preview.

+
The APIs page contains the collection of APIs in your portal and is known as the API Catalog. The template includes customizing the colors for each API type like REST and HTTP.

. Style the API Details page
+
The API Details page displays information about an API. The template includes the name, type of API, owner email address, and a description of the API. 

== Set Up an API Portal
Set up an API portal by choosing how to create the portal.

> . From the *Set up API portal* page of the guided setup, select an option to create the portal:

Reviewer: Aha, it is as I thought.
